apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/create-a-component.md

```diff
@@ -0,0 +1,290 @@
+   </Alert>
+
+   `INSERT SCREENSHOT OF completed form`
+
+1. Click on the **Save** button to complete creating the component definition. A message will display telling you that the component was created.
+
+1. Click on the **Click to start editing** link that is displayed at the top of the Component form. This will launch the canvas (or 'Section') view of the Text Teaser component where you can start adding elements in the builder.
+
+   `INSERT SCREENSHOT OF Canvas`
+
+### Create the column layout for the Text Teaser component
+
+Before we begin adding content, we want to ensure we can provide a two-column layout that matches the requirements that were given at the beginning. You will now learn how to create Card elements that can be used to hold other elements. For more details on creating layouts, you can take a look at our documentation on [Working with a component layout](https://doc.sitecore.com/xmc/en/users/xm-cloud/work-with-a-component-layout.html)
+
+1. Click on the section/canvas. You will see a grid displayed by default.
+
+   <Alert status="info">
+      <AlertIcon />
+      This can be adjusted on the pane at the right side of the screen. For now leave it as it is.
+   </Alert>
+
+   `INSERT SCREENSHOT of editor with Layout&Alignment section highlighted with a red box`
+
+1. Click and drag your mouse across the canvas to create a left column for your Text Teaser.
+
+   <Alert status="info">
+      <AlertIcon />
+      By default, XM Cloud will create a **Block** element. You will want to change this to a **Card** in the following steps.
+   </Alert>
+
+   `INSERT SCREENSHOT of Canvas with drawn Block area`
+
+1. Click on the newly created left column Block area to load the context bar above the area.
+
+1. Click on the element type **Block** in the context bar to change the element type.
+
+1. Select the **Card** element type from the pop-up menu.
+
+   `INSERT SCREENSHOT OF Block elements dialog`
+
+<Alert status="warning">
+   <AlertIcon />
+   **Challenge:** You will now create the right side of the column. Try to do it without the steps! If you get stuck, the full steps are here to help you through it.
+</Alert>
+
+1. Click and drag your mouse across the canvas to create a right column for your Text Teaser.
+
+   `INSERT SCREENSHOT of Canvas with drawn Block area`
+
+1. Click on the newly created right column Block area to load the context bar above the area.
+
+1. Click on the element type **Block** in the context bar to change the element type.
+
+1. Select the **Card** element type from the pop-up menu.
+
+### Add content elements to the columns
+
+Now that the layout is created, you will add content items to the columns to match the requirements provided. Based on the requirements, we need two headings on the left side (an H2 and an H3) and a block of free-form text on the right for the teaser content.
```

Comment:
you will add content items to the columns --> you will add text elements to the cards

Reply:
I've just pushed a change for this. Good catch!